src/main/java/seedu/address/logic/commands/GenerateEmailsCommand.java

```diff
@@ -23,15 +23,18 @@
             + "INDEX\n"
             + "Example: " + COMMAND_WORD + " 1";
 
-    static final String MESSAGE_NO_CONTACTS_ASSIGNED =
+    public static final String MESSAGE_NO_CONTACTS_ASSIGNED =
```

Comment:
intentionally used default access modifier as it is not used outside the package.

Reply:
I see i will revert it back then